tests/integration/cassettes/TestSubredditModeration.test_update.json

```diff
@@ -1,7 +1,7 @@
 {
   "http_interactions": [
     {
-      "recorded_at": "2020-04-05T00:47:30",
```

Comment:
Should this cassette be smaller since in theory it should have fewer requests?

Reply:
Nevermind. I see in the test (https://github.com/praw-dev/praw/blob/master/tests/integration/models/reddit/test_subreddit.py#L1610) that we fetch all settings before updating to compare afterwards.